src/components/search-form/search-form.css

```diff
@@ -73,6 +73,17 @@
   &:hover {
     background-color: color(var(--primary) shade(8%));
   }
+
+  &[disabled] {
+    background-color: #ccc !important;
+    border-color: #ccc !important;
+    color: #888 !important;
```

Comment:
I don't think  `!important` is needed on these three lines, since `.search-submit[disabled]` is more specific than the `.search-submit` that originally gave it background/border/text colors.

Reply:
Thanks @cherewaty - just made the correction and tried it out!